src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -473,14 +471,14 @@ protected function _bundleTranslatedFields($entity)
         }
 
         $results = $this->_findExistingTranslations($find);
-        $model = $this->config('conditions.model');
+        $alias = $this->_table->alias();
 
```

Comment:
OR why don't we use `$table->table()` - that's after all the fundamental "thing" that's being translated.